keras/backend/jax/core.py

```diff
@@ -314,3 +315,18 @@ def unstack(x, num=None, axis=0):
         jax.lax.index_in_dim(x, i, axis, keepdims=False)
         for i in range(x.shape[axis])
     ]
+
+
+@contextlib.contextmanager
+def device(device):
```

Comment:
Should we expose something along the lines of `list_devices` as part of this work? Seems like it would be useful to avoid needing to write backend specific code there, but maybe already considered.

https://github.com/keras-team/keras/blob/9c675a9a45e5e8244163fea82efc6066722608a1/keras/backend/jax/distribution_lib.py#L14

Reply:
+1

Reply:
Ack, this will be a bit more work, will defer to a future PR.